src/core/announcements/utilities.ts

```diff
@@ -0,0 +1,19 @@
+import { SignedAnnouncement } from "./crypto";
```

Comment:
minor but this seems more like something that belongs a in a service.

Reply:
any suggestion on where to put it instead?

Reply:
/service/....ts. or services.ts

Reply:
Updated